testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/clients/ClientScopeTest.java

```diff
@@ -62,7 +62,6 @@ public void before() {
     }
     
     @Test
-    @Ignore //phantomjs sometimes doens't enable "Add Selected >>" button when role is selected
```

Comment:
This is failing with Chrome. Luckily, there's an easy fix.
https://github.com/keycloak/keycloak/blob/14efa819bd2da1e5ffa55522b95a134f9e168815/testsuite/integration-arquillian/tests/other/console/src/main/java/org/keycloak/testsuite/console/page/roles/RoleCompositeRoles.java#L122
Change it to:
```java
roles.add(getTextFromElement(option).trim());
```

Reply:
Will fix it